crate/kmip/src/openssl/private_key.rs

```diff
@@ -755,6 +761,216 @@ mod tests {
         assert_eq!(private_key_.bits(), 253);
     }
 
+    #[test]
+    fn test_ec_ed25519_private_key() {
```

Comment:
those 4 tests are very similar, should we write a function?
- `test_ec_x25519_private_key`
- `test_ec_ed25519_private_key`
- `test_ec_x448_private_key`
- `test_ec_ed448_private_key`

Reply:
I propose commit c235c9b60ea8af0ada1dc9397fbe6495be25cf5f for a refacto of these tests. If it is accepted, I will use this pattern to refacto public key tests as well.

Reply:
commit 7f42f9ee61e3789c16fe7b55a33c5e34be132c94 should improve refacto and do it for public keys.